config/routes.rb

```diff
@@ -4,8 +4,9 @@
 
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  resources :posts
   resources :users
```

Comment:
Could we add resources :users do resources :posts end - Same as Users has many posts, posts belongs to users